chuffed/vars/int-var.h

```diff
@@ -251,7 +263,11 @@ class IntVar : public Var {
 	reverse_iterator rend() const { return reverse_iterator(begin()); }
 
 	int size() const {
+#ifdef HAS_VAR_IMPACT
+		if (!vals) return max - min;
```

Comment:
Why did you use "max - min" and not the "count" below?

Reply:
I wanted the option to be entirely independent from other options.

From my understanding, the current implementation of size can be called only if INT_DOMAIN_LIST, or if 'vals' is confirmed to be used, and will crash otherwise.

For instance, in IntVar::getScore, VAR_SIZE min and max return size based on max - min calculations, unless vals is defined.

The commented code is now outdated from fixing the off-by-one error.
As a side note, I would add that benchmark actually did not notice any slowing-down from adding the +1, so we could consider making VAR_SIZE scoring consistent.